code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -220,6 +220,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	var/list/spooky_scaries = list()
 	var/unlimited = 0
+	var/spec = /datum/species/skeleton
```

Comment:
You're missing auto-doc for this var, and the var name isn't super descriptive. maybe do something like applied_species